packages/core/package.json

```diff
@@ -75,6 +75,7 @@
     "asn1js": "^3.0.5",
     "aws-sdk": "^2.1243.0",
     "dayjs": "^1.11.9",
+    "formidable": "^3.5.1",
```

Comment:
https://www.npmjs.com/package/formidable

Reply:
mtom parsing library

Reply:
afaik multer is primarily used  for uploading files via html forms as **multipart/form-data**. Mtom is **multipart/related** data. 
Multer doesn't look like it supports parsing multipart/related data (no mentions of multipart/related in the codebase or in issues or examples), but formidable does look like it supports what we are using multer for. 

But just fyi, multer is only being used for internal/admin logic in our codebase. I can make that ticket but not sure its worth the effort of replacing it.